data/basal_test.go

```diff
@@ -0,0 +1,248 @@
+package data
+
+import (
+	"github.com/tidepool-org/platform/validate"
+
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/ginkgo"
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/gomega"
+)
+
+var _ = Describe("Basal", func() {
```

Comment:
I believe that the outer describe is the same on all the tests - when you do a `ginkgo generate basal` it will be created